### PR TITLE
feat(docs): 513 features en

### PR DESCRIPTION
**Pull request by Meggielqk**

### CODE REVIEW

File: en_US/api/deployment_manage.md [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+
+```json
+{
+    "connections": 1000,
+    "createAt": "2024-07-22 05:32",
+    "deploymentID": "w41b11c0",
+    "deploymentName": "deployment-w41b11c0",
+    "deploymentType": "dedicated",
+    "platform": "Alibaba Cloud",
+    "region": "Hangzhou",
```
Comment (Meggielqk) [inline en_US/api/deployment_manage.md:57]: Need to change the "platform" and "region" in en file.

File: en_US/billing/overview.md [hunk 2]
```diff
@@ -12,33 +12,43 @@ Billing accrues hourly and bills are generated monthly after the usage has occur
...
 Bill is issued monthly. In your Cloud Console, select **Billing** -> **Overview** from the top menu and you can see the details about your bill.
 
 The **Monthly Bill** pane displays the cumulative usage amount. To modify the billing period, locate the option in the top right corner of the pane. You can select a billing period ranging from the current month up to the previous 12 months. 
 
 The **Bill status** shows one of the following:
 
-- Pending: The monthly bill has not yet been billed.
-- Issued: The bill has been billed and paid.
-- Overdue: The bill has been billed and not paid.
+- `Pending`: The monthly bill has not yet been billed.
```
Comment (Meggielqk) [inline en_US/billing/overview.md:23]:
```suggestion
- **Pending**: The monthly bill has not yet been billed.
```

File: en_US/billing/overview.md [hunk 3]
```diff
@@ -12,33 +12,43 @@ Billing accrues hourly and bills are generated monthly after the usage has occur
...
+- `Issued`: The bill has been billed and paid.
```
Comment (Meggielqk) [inline en_US/billing/overview.md:24]:
```suggestion
- **Issued**: The bill has been billed and paid.
```

File: en_US/billing/overview.md [hunk 4]
```diff
@@ -12,33 +12,43 @@ Billing accrues hourly and bills are generated monthly after the usage has occur
...
+- `Overdue`: The bill has been billed and not paid.
```
Comment (Meggielqk) [inline en_US/billing/overview.md:25]:
```suggestion
- **Overdue**: The bill has been billed and not paid.
```

**Review by azuregaya — APPROVED** [review]
fixed and approved

**Pull request merged**